### DISCUSSION TITLE
Php

### DISCUSSION THREAD

**Discussion by rajisgreat** [community/community#37850] [Other Feature Feedback, Questions, & Ideas]
Is there no way to run a **Php** file even ?

**Comment by radar-blip** [comment]
I dont think you can as it's not supported.

Read the last line under Static site generators here:
https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#static-site-generators

**Comment by jfullstackdev** [comment]
GitHub Pages run only the core of static website plus its own features

it will not run server-side scripting because it's not designed for that, it's original intention is for documentation only which does not require talking to a server

but there are external services that can, say, make your blog comments possible but hopefully you can find one, I don't want to recommend,
